package/chainConfigs.ts

```diff
@@ -9,6 +9,11 @@ export const chainConfigs = {
     MINT_FEE_AMOUNT: 777000000000000,
     MINT_FEE_RECIPIENT: "0x7A810DCd0f8d83B20212326813Db6EF7E9FD030c",
   },
+  [11155111]: {
```

Comment:
maybe we should git ignore these files, and they are just built and bundled at publish time? same for the wagmi generated file

Reply:
yep good idea

Reply:
done!